### PR TITLE
[zh] Rework parallel processing expansion

### PR DESCRIPTION
**Pull request by tengqm**
The English content was COMPLETELY REWRITTEN for DAMN REASONS.

### CODE REVIEW

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 1]
```diff
@@ -1,341 +1,474 @@
...
+```
+
+<!--
+You can view `jobs.yaml` to verify that the `render_template` script worked
+correctly.
+
+Once you are happy that `render_template` is working how you intend,
+you can pipe its output into `kubectl`:
+-->
+你可以查看 `jobs.yaml` 以验证 `render_template` 脚本正常工作。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:352]:
```suggestion
你可以查看 `jobs.yaml` 以验证 `render_template` 脚本是否正常工作。
```

File: content/zh/docs/tasks/job/parallel-processing-expansion.md [hunk 2]
```diff
@@ -1,341 +1,474 @@
...
+objects.
+
+You could also consider writing your own [controller](/docs/concepts/architecture/controller/)
+to manage Job objects automatically.
+-->
+还有一些其他[作业模式](/zh/docs/concepts/workloads/controllers/job/#job-patterns)
+可供选择，这些模式都能用来处理大量任务而又不会创建过多的 Job 对象。
+
+你也可以考虑编写自己的[控制器](/zh/docs/concepts/architecture/controller/)
+来紫的管理 Job 对象。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/job/parallel-processing-expansion.md:473]:
```suggestion
来自动管理 Job 对象。
```

**Review by tanjunchen — COMMENTED** [review]
Overall good except for some comments.
/lgtm

**Comment by tengqm** [comment]
@tanjunchen All feedbacks addressed. Thanks for the careful review.

**Pull request merged**